versions/v0.9.1/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM node:4
```

Comment:
Ah, sorry about about `/tmp`. I was working on this late last night I guess I missed that.

I chose node 4 because it's LTS, though I'll admit that I'm using 6 on Arch Linux and haven't had any problem with it.